code/game/objects/structures/crates_lockers/closets/onestar.dm

```diff
@@ -0,0 +1,241 @@
+//Closets full of loot, they should be placed in derelicts
+
+//// Loot table on all tiers
+// Tier 1
+/obj/structure/closet/onestar/tier1
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, doesn't look like it contains anything worthwhile."
+	icon_state = "lootcloset"
+
+/obj/structure/closet/onestar/tier1/populate_contents()
+	new /obj/random/contraband/low_chance(src)
+	new /obj/random/contraband/low_chance(src)
+	new /obj/random/pack/rare/low_chance(src)
+	new /obj/random/junk(src)
+	new /obj/random/junk(src)
+	new /obj/random/junk(src)
+	new /obj/random/lowkeyrandom(src)
+	new /obj/random/lowkeyrandom(src)
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/pack/tech_loot/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/gun_loot/low_chance(src)
+
+// Tier 2
+/obj/structure/closet/onestar/tier2
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, there's good stuff inside it."
```

Comment:
Again, the characters don't know what is inside, they might only assume. 
"Old OneStar closet, it looks like there might be something good inside." or something along those lines.
They are essentially lootboxes and you don't know what's inside of them, before you actually open them.
Same goes for the tier 3, as the characters have no clue what's inside, they can only see it looks like something that people would shove valuables into.